jetty-client/src/main/scala/org/http4s/client/jetty/JettyClient.scala

```diff
@@ -4,141 +4,64 @@ package jetty
 
 import cats.data.Kleisli
 import cats.effect._
-import cats.effect.implicits._
 import cats.implicits.{catsSyntaxEither => _, _}
 import fs2._
-import fs2.Stream._
-import java.nio.ByteBuffer
 import org.eclipse.jetty.client.HttpClient
-import org.eclipse.jetty.client.api.{Result, Request => JettyRequest, Response => JettyResponse}
-import org.eclipse.jetty.client.util.DeferredContentProvider
-import org.eclipse.jetty.http.HttpFields
-import org.http4s.internal.loggingAsyncCallback
-import org.log4s.getLogger
-import scala.collection.JavaConverters._
+import org.eclipse.jetty.client.api.{Request => JettyRequest}
+import org.eclipse.jetty.http.{HttpVersion => JHttpVersion}
+import org.log4s.{Logger, getLogger}
 import scala.concurrent.ExecutionContext
-import scala.util.control.NonFatal
 
 object JettyClient {
 
-  private val logger = getLogger
+  implicit private val logger: Logger = getLogger
 
   def apply[F[_]](client: HttpClient = new HttpClient())(
       implicit F: ConcurrentEffect[F],
       ec: ExecutionContext): F[Client[F]] =
     F.pure(client)
       .flatTap(client => F.delay { client.start() })
-      .map(client => {
+      .map(client =>
         Client(
           Kleisli { req =>
-            F.async[DisposableResponse[F]] { cb =>
-              toRequest(client, req).send(responseHandler(cb))
+            F.asyncF[DisposableResponse[F]] { cb =>
+              Resource
```

Comment:
You could just use `Bracket` here instead of making a `Resource`.  `Bracket` excels at nesting more than anything.